signer.js

```diff
@@ -1,32 +1,101 @@
 'use strict'
 
```

Comment:
@mcollina 
I would like to modify the header like this.

```suggestion

// Inspired by node-cookie-signature
// https://github.com/tj/node-cookie-signature
//
// The MIT License
// Copyright (c) 2012–2022 LearnBoost <tj@learnboost.com> and other contributors;

```